src/qt/bitcoingui.cpp

```diff
@@ -462,6 +462,15 @@ void BitcoinGUI::createToolBars()
         toolbar->addAction(receiveCoinsAction);
         toolbar->addAction(historyAction);
         overviewAction->setChecked(true);
+
+#ifdef ENABLE_WALLET
```

Comment:
this `#ifdef` could be avoided/removed, right?

Reply:
Maybe.

Reply:
Has to bee there I guess because of `bool setCurrentWallet(const QString& name);` being only defined if `ENABLE_WALLET`.